src/Views/Shared/_Layout.cshtml

```diff
@@ -94,9 +94,8 @@
         <p class="govuk-phase-banner__content">
           <strong class="govuk-tag govuk-phase-banner__content__tag">BETA</strong>
           <span class="govuk-phase-banner__text">
-            This is a new service – your
-            <a href="mailto:becomingateacher@digital.education.gov.uk?subject=Search%20beta%20feedback" class="govuk-link">feedback</a>
-            will help us to improve it.
+            Tell us what you think of Find through a
```

Comment:
Can we change this to:

This is a new service – Tell us what you think, [take a short survey to give us your feedback]()

Reply:
Nice. Thanks